src/commands/approval.yml

```diff
@@ -41,6 +41,12 @@ steps:
           exit 1
         fi
 
+  - run:
+      name: Provide error if cURL is not installed.
+      command: |
+        which curl > curl_exists; echo $? | grep -q '1' && echo curl not installed && rm curl_exists && exit 1
```

Comment:
@piyush-insider try implementing this it looks better and without any External file creation. https://scripter.co/check-if-a-command-exists-from-shell-script/

Reply:
Sure. Will take a look at it

Reply:
It looks as if once this change is added we should be ready to merge.

Reply:
`command -v` should be a good way to do it too, I think.